.travis.yml

```diff
@@ -19,9 +19,9 @@ matrix:
     - jdk: "oraclejdk7" # this will also deploy a uberjar to s3 at some point
       env: PROFILE="-Dhadoop.profile=1"
     - jdk: "openjdk7"
-      env: PROFILE="-P!include-yarn -Dhadoop.version=2.0.0-alpha"
-    - jdk: "oraclejdk7"
-      env: PROFILE="-Dhadoop.version=2.2.0"
+      env: PROFILE="-Dhadoop.version=2.4.0"
```

Comment:
I am okay, but this is a pretty important thing. Maybe we should wait for at least one more committer agreeing with us.